homeassistant/components/adguard/services.yaml

```diff
@@ -16,10 +16,19 @@ add_url:
       example: https://www.example.com/filter/1.txt
       selector:
         text:
+    list_type:
+      name: List type
+      description: The list type (blocklist/allowlist) of the filter you are adding.
+      default: blocklist
+      selector:
+        select:
+          options:
```

Comment:
I think this should match the `LIST_TYPES`

Reply:
you are right...